src/components/forms/badge-form.js

```diff
@@ -90,7 +96,9 @@ class BadgeForm extends React.Component {
             }
         };
 
-        const badge_type_ddl = currentSummit.badge_types.map(bt => ({label: bt.name, value: bt.id}));
+        const badge_type_ddl = currentSummit.badge_types.map(bt => ({ label: bt.name, value: bt.id }));
+
+        const badge_view_type_ddl = currentSummit.badge_types.find(bt => bt.id === entity.type_id).allowed_view_types.map(vt => ({ label: vt.name, value: vt.id }));
```

Comment:
we need to allow also the ALL option
so when user select all, the view type is not send to the print app